.github/workflows/lint.yml

```diff
@@ -0,0 +1,99 @@
+# SPDX-License-Identifier: Apache-2.0
+
+name: Lint
+
+on:
+  push:
+    branches:
+      - "main"
+      - "release-**"
```

Comment:
Will we be using release branches?

Reply:
I created a branch filter in the org settings to allow for release branches. I'm not sure if we want to use release branches at this very moment (because it's too early to tell), but if we're going to add versioning for different CI actions, it might be useful to have release branches.

I can definitely remove release branches for now though. 👍